neuronsimulator/templates/neuronsimulator/plot_neural_responses.html

```diff
@@ -0,0 +1,140 @@
+{% extends "base.html" %}
+{% load static %}
+{% block title %}Neuron Responses{% endblock %}
+
+{% block head_extras %}
+    {{ block.super }}
+    <link href="{% static 'vanillaSelectBox/vanillaSelectBox.css' %}" rel="stylesheet">
+
+    <style>
+        .select-zone {
+            display: block;
+            width: 100%;
+            min-height: 200px;
+        }
+        label {
+            display: inline-block;
+            min-width: 300px;
+            width: 300px;
+        }
+    </style>
+{% endblock %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-6">
+            <h5>Select or enter input parameters for plotting neural responses:</h5>   
+            <br>
+            <div class="select-zone">
+                <label>Select a stimulated neuron:</label>
+                <select id="one-neuron">
+                    {% for neuron in neurons %}
+                        <option value="{{ neuron.name }}">{{ neuron.name }}</option>
+                    {% endfor %}
+                </select>
+            </div>
+
+            <div class="select-zone">
+                <label>Select one or more neuron(s) for calculating responses:</label>
+                <select id="multi-neurons" multiple="" size="12" style="display: inline-block;">
+                    {% for neuron in neurons %}
+                        <option value="{{ neuron.name }}">{{ neuron.name }}</option>
+                    {% endfor %}
+                </select>
```

Comment:
Q. Why are these select lists not located inside `form` tags? Are the form tags in a parent template or are these used in javascript?